app/code/Magento/Review/Block/Product/ReviewRenderer.php

```diff
@@ -30,32 +33,21 @@ class ReviewRenderer extends \Magento\Framework\View\Element\Template implements
     ];
 
     /**
-     * Review model factory
-     *
-     * @var \Magento\Review\Model\ReviewFactory
+     * @var ReviewSummaryData
      */
-    protected $_reviewFactory;
+    private $reviewSummaryData;
 
     /**
-     * @var ReviewSummaryFactory
-     */
-    private $reviewSummaryFactory;
-
-    /**
-     * @param \Magento\Framework\View\Element\Template\Context $context
-     * @param \Magento\Review\Model\ReviewFactory $reviewFactory
+     * @param Context $context
+     * @param ReviewSummaryData $reviewSummaryData
      * @param array $data
-     * @param ReviewSummaryFactory $reviewSummaryFactory
      */
     public function __construct(
-        \Magento\Framework\View\Element\Template\Context $context,
-        \Magento\Review\Model\ReviewFactory $reviewFactory,
-        array $data = [],
-        ReviewSummaryFactory $reviewSummaryFactory = null
+        Context $context,
+        ReviewSummaryData $reviewSummaryData,
+        array $data = []
```

Comment:
Please, make changes according to the BC development guide.

Reply:
Done